src/mrack/data/mrack.conf

```diff
@@ -0,0 +1,4 @@
+[mrack]
+mrackdb = .mrackdb.json
+provisioning-config = provisioning-config.yaml
+
```

Comment:
Could you please add here `metadata` key value example?

Reply:
> Mrack looks for the config on these locations:
>
>    user provided path via -c or --mrack-config global option
>    ~./mrack.conf

I believe `~` should not be there

Reply:
Commit message ammended.

Reply:
And metadata added to data/mrack.conf